python/sglang/srt/model_executor/forward_batch_info.py

```diff
@@ -18,10 +18,11 @@
 """Meta data for a forward pass."""
 from dataclasses import dataclass
 from enum import IntEnum, auto
-from typing import TYPE_CHECKING, List
+from typing import TYPE_CHECKING, List, Tuple
 
 import numpy as np
 import torch
+from vllm.model_executor.layers.rotary_embedding import MRotaryEmbedding
```

Comment:
It seems this class is python only without c++ custom ops. Can you directly copy this class under `srt/layers/rotary_embedding.py`?

Reply:
done